src/__tests__/hasSubArray.test.ts

```diff
@@ -0,0 +1,21 @@
+import { hasSubArray } from '../helperFunctions/hasSubArray';
+
+describe('hasSubArray', () => {
+  it('will find a sub-array in a master array', () => {
+    const array = [[0, 0], [0, 1], [0, 2]];
+    const subArray = [0, 0];
+    expect(hasSubArray(array, subArray)).toBe(true);
+  });
+
+  it('will not send a false positive', () => {
```

Comment:
> `'will not send a false positive'`
is hard for me to understand at first glance. What about something like "will return false if subArray not found" etc?